lib/tasks/dev.rake

```diff
@@ -29,6 +29,26 @@ namespace :dev do
     install_phantomjs
   end
 
+  desc 'Toggle user permission for category admin'
+  task :toggle_category, [:email] => :environment do |_t, args|
```

Comment:
this should be called category_admin and should work like the user_admin task.  IE. email is provided as a env var and if none is provided it picks the last user.